library/src/blas1/rocblas_axpy.cpp

```diff
@@ -135,33 +248,150 @@ rocblas_axpy_template(rocblas_handle handle,
     else if(nullptr == handle)
         return rocblas_status_invalid_handle;
 
-    /*
-     * Quick return if possible. Not Argument error
-     */
-    if ( n <= 0 )
+    if (n <= 0) // Quick return if possible. Not Argument error
+    {
         return rocblas_status_success;
+    }
 
-    int blocks = (n-1)/ NB_X + 1;
+    int blocks = ((n-1) / NB_X) + 1;
 
-    dim3 grid( blocks, 1, 1 );
+    dim3 grid(blocks, 1, 1);
     dim3 threads(NB_X, 1, 1);
 
     hipStream_t rocblas_stream;
     RETURN_IF_ROCBLAS_ERROR(rocblas_get_stream(handle, &rocblas_stream));
 
-    if( rocblas_pointer_to_mode((void*)alpha) == rocblas_pointer_mode_device ){
-        hipLaunchKernel(HIP_KERNEL_NAME(axpy_kernel_device_scalar), dim3(blocks), dim3(threads), 0, rocblas_stream, n, alpha, x, incx, y, incy);
+    if (rocblas_pointer_mode_device == rocblas_pointer_to_mode((void*)alpha))
+    {
+        hipLaunchKernel(HIP_KERNEL_NAME(axpy_kernel_device_scalar), 
```

Comment:
I think we are supposed to start using the newew API: [hipModuleLaunchKernel ](https://github.com/ROCm-Developer-Tools/HIP/blob/9f4ea9255cb44af249d156204b60b6a7fa206143/docs/markdown/hip_porting_driver_api.md#using-hipmodulelaunchkernel)

Lets stop adding more of the deprecated hipLaunchKernel.  This comment applies to the rest of the file

Reply:
Will refactor in a separate pull request to use new API